### PR TITLE
Moving single-spa-angular@3 docs to single-spa.js.org

### PR DESCRIPTION
**Pull request by internettrans**
The single-spa-angular docs are getting unwieldy within the single-spa-angular README. Now that single-spa-angular@3.0.0-rc.0 is out and we're close to an official 3.0.0 release, I decided to put some  effort into creating good docs for single-spa-angular.

Having them here in single-spa.js.org gives us a nice right sidebar for people to jump around through the (extensive) documentation.

single-spa-angular-docs

### CODE REVIEW

File: docs/ecosystem.md [hunk 1]
```diff
@@ -11,15 +11,15 @@ There is a growing number of projects that help you bootstrap, mount,
 and unmount your applications that are written with popular frameworks. Feel free
 to contribute to this list with your own project:
 
-- [single-spa-angularjs](./ecosystem-angularjs.md)
+- [single-spa-react](./ecosystem-react.md)
```
Comment (internettrans) [inline docs/ecosystem.md:14]: Just rearranged based on popularity.

File: docs/ecosystem-angular.md [hunk 2]
```diff
@@ -3,8 +3,454 @@ id: ecosystem-angular
...
+
+### Angular 2
+Angular 2 is supported by single-spa-angular.
+
+The [single-spa-angular schematics](#schematics) are not supported by Angular 2, so you'll have to
+follow the [steps for manual installation](#manual-installation). The
+[single-spa helpers](#the-single-spa-helpers) work with Angular 2.
+
+### Angular 3
+Angular 3 [never existed](https://www.infoworld.com/article/3150716/forget-angular-3-google-skips-straight-to-angular-4.html).
```
Comment (frehner) [inline docs/ecosystem-angular.md:41]: Haha I love the angular world

File: docs/ecosystem-angular.md [hunk 3]
```diff
@@ -3,8 +3,454 @@ id: ecosystem-angular
...
+
+The single-spa-angular schematics perform the following tasks:
+* Install single-spa-angular.
+* Generate a `main.single-spa.ts` in your project `src/`.
+* Generate `single-spa-props.ts` in `src/single-spa/`
+* Generate `asset-url.ts` in `src/single-spa/`
+* Generate an EmptyRouteComponent in `src/app/empty-route/`, to be used in app-routing.module.ts.
+* Add an npm script `npm run build:single-spa`.
+* Add an npm script `npm run serve:single-spa`.
+* For Angular 7 only, create a new entry in the project's architect called `single-spa`, which is
```
Comment (frehner) [inline docs/ecosystem-angular.md:112]: Just to verify, this is only for 7? Or >= 7

**Review by frehner — APPROVED** [review]
Not gonna lie, I tried to read it all but i didn’t understand it all. Small question but otherwise LGTM

File: docs/ecosystem-angular.md (hunk 3 above)
Comment (internettrans) [reply docs/ecosystem-angular.md:112]: Just Angular 7, because Angular 8 does it in a different way with @angular-builders/custom-webpack

**Pull request merged**